test/Waives.Reactive.Tests/TestDocument.cs

```diff
@@ -5,7 +5,7 @@ namespace Waives.Reactive.Tests
 {
     internal class TestDocument : Document
     {
-        public TestDocument(byte[] contents) : base("Test Document")
+        public TestDocument(byte[] contents, string sourceId = "Test Document") : base(sourceId)
```

Comment:
I think this change can be dropped, I'm not sure we're using it anywhere at the moment?